Gemfile

```diff
@@ -1,8 +1,7 @@
 # frozen_string_literal: true
-
 source 'https://rubygems.org'
 
-ruby '2.6.5'
+ruby File.open('.ruby-version').readline.chomp
```

Comment:
Too much as for me, we don't change ruby-version so frequently to use this thing, but as you wish ;)